docs/reference/camera.md

```diff
@@ -180,9 +180,13 @@ So this mode is costly in terms of performance! Reducing the resolution of the c
 
 When the camera is spherical, the image returned by the `wb_camera_get_image` function is a 2-dimensional array (s,t) in spherical coordinates.
 
-Let `hFov` be the horizontal field of view, and let `theta` be the angle in radian between the `(0, 0, -z)` relative coordinate and the relative coordinate of the target position along the `xz` plane relative to the camera, then `s=0` corresponds to a `theta` angle of `-hFov/2`, `s=(width-1)/2` corresponds to a `theta` angle of 0, and `s=width-1` corresponds to a `theta` angle of `hFov/2`.
+Let assume a 3D target point is located at coordinates `(X, Y, Z)` relative to the camera origin.
 
-Similarly, let `vFov` be the vertical field of view (defined just above), and `phi` the angle in radian between the `(0, 0, -z)` relative coordinate and the relative coordinate of the target position along the `xy` plane relative to the camera, `t=0` corresponds to a `phi` angle of `-vFov/2`, `t=(height-1)/2` corresponds to a `phi` angle of 0, and `t=height-1` corresponds to a `phi` angle of `vFov/2`).
+Let `hFov` be the horizontal field of view, and let `theta` be the angle in radian between the vector to `(X, 0, 0)` and the vector to `(X, Y, 0)`, i.e., the projection of the target point position along the `xy` plane relative to the camera.
```

Comment:
Honestly, this is simpler and clearer to me:
```suggestion
Let `hFov` be the horizontal field of view, and let `theta` be the angle in radian between vector `(X, 0, 0)` and vector `(X, Y, 0)`.
```
@BenjaminDeleze & @ygoumaz: what do you think?

Reply:
I think this version is better. Calling a vector by its components is totally correct and better than using the formulation 'to <coordinates>'. Also I believe the part of the sentence about the projection is a bit complicated and not very understandable.  I would also remove it.

However, I would add a simple diagram containing the camera, a sphere, the point (X,Y,Z) and the 2 angles `theta` and `phi`, because any textual formulation here would be complicated to really understand what is going on.